deposit/src/main/resources/fcrepo-clients-context.xml

```diff
@@ -48,8 +48,15 @@
 		<property name="aclService" ref="aclService" />
 	</bean>
 	
-	<bean id="repository" class="edu.unc.lib.dl.fcrepo4.Repository">
+	<bean id="repositoryObjectCacheLoader" class="edu.unc.lib.dl.fcrepo4.RepositoryObjectCacheLoader">
 		<property name="client" ref="fcrepoClient" />
+		<property name="repository" ref="repository" />
```

Comment:
wouldn't  the repository bean need to be defined before the repositoryObjectCacheLoader, since the latter refers to the former?